toolkit/resources/manifests/package/toolchain_aarch64.txt

```diff
@@ -114,8 +114,8 @@ gdbm-1.18-4.cm2.aarch64.rpm
 gdbm-debuginfo-1.18-4.cm2.aarch64.rpm
 gdbm-devel-1.18-4.cm2.aarch64.rpm
 gdbm-lang-1.18-4.cm2.aarch64.rpm
-gettext-0.21-1.cm2.aarch64.rpm
-gettext-debuginfo-0.21-1.cm2.aarch64.rpm
+gettext-0.21-2.cm2.aarch64.rpm
```

Comment:
Please also update `toolkit/resources/manifests/package/update_manifests.sh`. The new subpackages should be mentioned inside the `toolchain_*txt` manifests but I believe they are unnecessary inside `pkggen_core_*.txt`. Running the script should yield such results. This means that you'll need to update the `remove_packages_for_pkggen_core` function to remove filter out the new subpackages.